app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ReviewsController < ApplicationController
+
+
+  def new
+    @shelter = Shelter.find(params[:shelter_id])
+  end
+
+  def create
+    shelter = Shelter.find(params[:shelter_id])
+    review = shelter.reviews.new(review_params)
+    review.save
```

Comment:
Looks like you solved why our reviews weren't showing up in the database.  It will also be useful for the flash notice.

Reply:
Thank you for the feedback!